Jenkinsfile

```diff
@@ -1,89 +1 @@
-
-            junit testResults: '**/target/*-reports/TEST-*.xml'
-
-            recordIssues enabledForFailure: true, tool: mavenConsole(), referenceJobName: 'Plugins/warnings-ng-plugin/master'
-            recordIssues enabledForFailure: true, tools: [java(), javaDoc()], sourceCodeEncoding: 'UTF-8', referenceJobName: 'Plugins/warnings-ng-plugin/master'
-            recordIssues enabledForFailure: true, tool: checkStyle(pattern: 'target/checkstyle-result.xml'), sourceCodeEncoding: 'UTF-8', referenceJobName: 'Plugins/warnings-ng-plugin/master'
-            recordIssues enabledForFailure: true, tool: cpd(pattern: 'target/cpd.xml'), sourceCodeEncoding: 'UTF-8', referenceJobName: 'Plugins/warnings-ng-plugin/master'
-            recordIssues enabledForFailure: true, tool: pmdParser(pattern: 'target/pmd.xml'), sourceCodeEncoding: 'UTF-8', referenceJobName: 'Plugins/warnings-ng-plugin/master'
-            recordIssues enabledForFailure: true, tool: spotBugs(pattern: 'target/spotbugsXml.xml'), sourceCodeEncoding: 'UTF-8', referenceJobName: 'Plugins/warnings-ng-plugin/master'
-            recordIssues enabledForFailure: true, tool: taskScanner(includePattern:'**/*.java', excludePattern:'target/**/*,**/TaskScannerTest*', highTags:'FIXME', normalTags:'TODO'), sourceCodeEncoding: 'UTF-8', referenceJobName: 'Plugins/warnings-ng-plugin/master'
-            jacoco()
-            sh "curl -s https://codecov.io/bash | bash -s - -t c4071f73-a222-43ff-a41b-a6c8c118e242"
-        }
-    }
-}
-
-node ('windows') {
-    timeout(200) {
-        stage ('Windows Checkout') {
-            checkout scm
-        }
-
-        stage ('Windows Build') {
-            String jdk = '8'
-            String jdkTool = "jdk${jdk}"
-            List<String> env = [
-                    "JAVA_HOME=${tool jdkTool}",
-                    'PATH+JAVA=${JAVA_HOME}/bin',
-            ]
-            String command
-            List<String> mavenOptions = [
-                    '--batch-mode',
-                    '--errors',
-                    '--update-snapshots',
-                    '-Dmaven.test.failure.ignore',
-            ]
-            if (jdk.toInteger() > 7 && infra.isRunningOnJenkinsInfra()) {
-                /* Azure mirror only works for sufficiently new versions of the JDK due to Letsencrypt cert */
-                def settingsXml = "${pwd tmp: true}/settings-azure.xml"
-                writeFile file: settingsXml, text: libraryResource('settings-azure.xml')
-                mavenOptions += "-s $settingsXml"
-            }
-            mavenOptions += "clean verify -Djenkins.test.timeout=1000"
-            command = "mvn ${mavenOptions.join(' ')}"
-            env << "PATH+MAVEN=${tool 'mvn'}/bin"
-
-            withEnv(env) {
-                bat command
-            }
-
-            junit testResults: '**/target/*-reports/TEST-*.xml'
-        }
-    }
-}
+buildPlugin(useAci: true, platforms: ['maven'])
```

Comment:
```suggestion
buildPlugin(useAci: true)
```

Reply:
You need to delete the `platforms` here (or use `linux`). That is why your build is still using `tool` rather than picking up the new version of Maven packaged in the ACI image. See: https://github.com/jenkins-infra/pipeline-library/blob/master/vars/buildPlugin.groovy#L41-L42